etc/shadow-maint/groupdel-pre.d/01-kill_group_procs.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+GROUPID=`awk -F: '$1 == "'"${SUBJECT}"'" { print $3 }' /etc/group`
```

Comment:
What happens if the user is not in /etc/group ? Does that matter?
The script would currently just not kill any processes.

An existing user might not be in /etc/group due to libnss-extrausers or yellow-pages or something like ldap integration.